gcloud/pubsub/_gax.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+try:
+    from google.gax import CallOptions
+except ImportError:
+    _HAVE_GAX = False
+else:
+    _HAVE_GAX = True
+
+
+    from google.gax.errors import GaxError
```

Comment:
I'm pretty sure I don't understand you:  there is no point in having the `gcloud.pubsub._gax._PublisherAPI` class even exist if GAX is not importable.